src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -423,16 +423,6 @@ public IEnumerable<OneWayCompatibilityMappingEntry> CompatibilityMappings
                                 FrameworkConstants.CommonFrameworks.NetStandard15),
 
                             CreateGenerationAndStandardMappingForAllVersions(
-                                FrameworkConstants.FrameworkIdentifiers.XamarinIOs,
```

Comment:
This is a minor cleanup. The deleted lines are duplicate of 420 - 423. I can make a separate PR if you'd like.